flow-typed/box-ui-elements.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @flow
+ * @file Flow types
+ * @author Box
+ */
+/* eslint-disable no-use-before-define */
+import type { MessageDescriptor, InjectIntlProvidedProps } from 'react-intl';
+
+type BoxItemPermission = {
+    can_comment?: boolean,
```

Comment:
Why not use AnnotationPermissions? Those are the only permissions we care about right?

Reply:
I don't think i need this set of BoxItemPermissions because i added my own subset of that in box-annotations.js
```
type FilePermissions = {
    can_annotate: boolean,
    can_view_annotations_all: boolean,
    can_view_annotations_self: boolean
};
```

Reply:
AnnotationPermissions is specific to each individual annotation whereas there are a set of permissions which are specific to the file version as well